modules/autocomplete-picker/main/index.coffee

```diff
@@ -224,4 +224,4 @@ hx.autocompletePicker = (items, options) ->
   new AutocompletePicker(selection.node(), items, options)
   selection
 
-hx.AutocompletePicker = AutocompletePicker
\ No newline at end of file
+hx.AutocompletePicker = AutocompletePicker
```

Comment:
These new-lines at the end of a file are IntelliJ's default addition, which I don't think is a bad idea